src/pybind/mgr/rgw/module.py

```diff
@@ -349,3 +353,13 @@ def shutdown(self) -> None:
         self.log.info('Stopping')
         self.run = False
         self.event.set()
+
+    def import_realm_token(self,
+                           zone_name: Optional[str] = None,
+                           realm_token: Optional[str] = None,
+                           port: Optional[int] = None,
+                           placement: Optional[str] = None,
+                           start_radosgw: Optional[bool] = True,
+                           zone_endpoints: Optional[str] = None) -> None:
+        self._cmd_rgw_zone_create(zone_name, realm_token, port, placement, start_radosgw,
+                                  zone_endpoints)
```

Comment:
In this call I'm afraid the decorators `@check_orchestrator` and `@CLICommand('rgw zone create', perm='rw')` both of them are not necessary in you case I think. I'd suggest create a new function `rgw_zone_create` and call it both from the `_cmd_rgw_zone_create` and from your dashboard code.

Reply:
Thanks @rkachach , updated the PR